test/locations.jl

```diff
@@ -10,5 +10,11 @@ using DataDeps: try_determine_load_path, determine_save_path, try_determine_pack
         @test try_determine_package_datadeps_dir(fn) == nothing
     end
 
-end
+    mktemp() do fn, fh
+        ENV["DATADEPS_LOAD_PATH"] = ""
+        ENV["DATADEPS_NO_STANDARD_LOAD_PATH"] = false
+        @test try_determine_package_datadeps_dir(fn) == nothing
```

Comment:
Why is this test being added?
I'm not sure I see the connection to #89 

At a guess maybe it was mistakenly duplicated from above?

Reply:
Yeah, I was just checking that my method of testing locally was actually testing the right thing by adding an extra test that I knew worked, Removed now